DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserCardDataService.cs

```diff
@@ -7,41 +7,43 @@
 
     public partial class UserDataService
     {
+        private const string DelegateUserSelectQuery = @"cd.CandidateID AS Id,

```

Comment:
I would go as far as to include all the other common aspects that are shared for this query:
- `SELECT `
- `FROM Candidates AS cd
  INNER JOIN Centres AS ct ON ct.CentreID = cd.CentreID
  INNER JOIN JobGroups AS jg ON jg.JobGroupID = cd.JobGroupID`

Reply:
I have added this to the constant.